### DISCUSSION TITLE
Эффект "Рыбки СССР"

### DISCUSSION THREAD

**Discussion by Slovator** [vortigont/FireLamp_JeeUI#130] [Ideas]
Было бы превосходно, если бы добавили в проект эффект плавающих рыбок, как в советском светильнике. Видео с демонстрацией работы старого светильника прилагаю по ссылке. 
https://youtu.be/GEYPF0x3qFw?si=BnsD0FzJxeInvMVC

**Comment by vortigont** [comment]
матрицы на светодиодах ws2812 слишком крупные, "рыбки" не получатся. Для хаб панелей не получится сделать тонкую трубу. Вроде как есть гибкие матрицы на перспективу, сделать спрайт из рыбок и двигать его по матрице.

**Comment by Slovator** [comment]
Тестировал прошивку FieryLedLamp, там есть в наборе эффект "Планета Земля".... Так вот на матрице 16x16 через рассеивающий плафон (33 идеи) вполне различимы материки. Поэтому в принципе для теста можно попробовать пустить по кругу желтые и красные пятнышки на синем фоне.

**Comment by vortigont** [comment]
можно попробовать. Сейчас у проекта нет людей которые занимались бы эффектами - наводили красоту, тестировали, подбирали параметры и пр.
Пока идеи по эффектам складываем в "долгий ящик".
